pkgs/sketch_pad/README.md

```diff
@@ -16,5 +16,5 @@ This is an experimental re-imagining of the DartPad UI. Goals include:
 To run this locally, run:
 
 ```
-flutter run -d chrome
+flutter run -d chrome --web-browser-flag "--disable-web-security"
```

Comment:
What does this flag do?

Reply:
This flag tells chrome to not require that the server use the correct CORS headers for loading some of the flutter web resources (`AssetManifest.json`, `FontManifest.json`, ...). We could probably also address this is the web server used by `flutter run -d chrome` had some config options that would enable us to set CORS headers for some files.

In any case, I'll document this in the readme.